stl/inc/xutility

```diff
@@ -5284,9 +5284,75 @@ _NODISCARD constexpr _Iter_diff_t<_InIt> _Count_pr(_InIt _First, const _InIt _La
     return _Count;
 }
 
+enum class _TrimResult : unsigned char { _Continue, _Break, _ReturnFalse, _ReturnTrue };

```

Comment:
This enum and its enumerators are nice and self-explanatory - however, after looking at its usage, I recommend renaming `_Continue`. That's because all of the other enumerators literally say what should happen - `_Break` means we're going to `break;` etc. However, `_Continue` doesn't mean to `continue;`, it means to "keep going". If this were named `_KeepGoing` or something like that, then any potential for confusion would be avoided.